mozilla-release/browser/branding/branding-common.mozbuild

```diff
@@ -19,16 +19,10 @@ def FirefoxBranding():
             'VisualElements_70.png',
         ]
         BRANDING_FILES += [
-<<<<<<< HEAD
             'appname.bmp',
             'bgintro.bmp',
-            'bgstub.bmp',
-||||||| merged common ancestors
-            'bgstub.bmp',
-=======
             'bgstub.jpg',
```

Comment:
what for those are used for? should we have cliqz branded versions?

Reply:
For now we use stub_old.nsi script to generate our mini_installer. Later, of course, we will need to make our own changes here, of course.